components/meta-packages/history/history

```diff
@@ -661,6 +661,7 @@ library/apr-util/dbd-freetds@1.5.4,5.11-2020.0.1.6
 library/arcus@2.7.0,5.11-2023.0.0.3
 library/audio/audiofile@0.2.7,5.11-2022.1.0.3
 library/audio/gstreamer/plugin/gnonlin@0.10.17,5.11-2022.1.0.2
+library/ruby/bundler@1.13.6-2020.0.1.1
 library/c++/net6@1.3.14,5.11-2014.1.3.0
```

Comment:
The latest bundler is alread at
library/ruby/bundler@1.13.6-2025.0.0.1:20250314T144332Z
So you need to increment it to -2025.0.0.2